packages/web/src/services/audius-backend/Explore.ts

```diff
@@ -132,6 +138,21 @@ class Explore {
     }
   }
 
+  static async getMostLovedTracks(userId: ID, limit = 25) {
+    try {
+      const encodedUserId = encodeHashId(userId)
+      const tracks: APITrack[] = await libs().discoveryProvider.getMostLovedTracks(
+        encodedUserId,
+        limit,
+        true
+      )
+      return tracks.map(adapter.makeTrack).filter(removeNullable)
+    } catch (e) {
+      console.error(e)
+      return []
```

Comment:
just wondering what the error case is here, i guess seems like the pattern in the file?

Reply:
Retuning `[]` causes the page to show zero tracks in this playlist - I'm not sure what the intended error behavior is.